software/base/src/main/java/brooklyn/event/feed/jmx/JmxFeed.java

```diff
@@ -198,6 +198,7 @@ protected JmxFeed(Builder builder) {
         
         SetMultimap<String, JmxAttributePollConfig<?>> attributePolls = HashMultimap.<String,JmxAttributePollConfig<?>>create();
         for (JmxAttributePollConfig<?> config : builder.attributePolls) {
+            if (!config.isEnabled()) continue;
```

Comment:
These are tricky. Not sure I follow your suggestion of "adding `builder.enabledPolls`"? If we put the config into `attributePolls`, then `preStart()` would need to do the filtering. For example, in `registerAttributePoller` it would need to guard the call to `getPoller().scheduleAtFixedRate` if they were all disabled and ensure the handler(s) didn't get called. It would also need to set minPeriod appropriately (i.e. miss out the value of disabled poll configs).